### PR TITLE
kraken: Loading of realtime data from kirin

### PR DESCRIPTION
**Pull request by kinnou02**

### CODE REVIEW

File: source/kraken/maintenance_worker.cpp [hunk 1]
```diff
@@ -60,6 +60,41 @@ void MaintenanceWorker::load(){
...
     }
+
+    load_realtime();
+}
+
+
+void MaintenanceWorker::load_realtime(){
+    LOG4CPLUS_INFO(logger, "loading realtime data");
+
+    std::string queue_name = channel->DeclareQueue("", false, false, true, true);
```
Comment (xlqian) [inline source/kraken/maintenance_worker.cpp]: hmmm, the queue is created, but i don't see it's bound to any exchange with any routing key, is that normal?

File: source/kraken/configuration.cpp [hunk 2]
```diff
@@ -110,6 +112,14 @@ int Configuration::nb_thread() const{
     return this->vm["GENERAL.nb_threads"].as<int>();
 }
 
+bool Configuration::enable_realtime() const{
```
Comment (xlqian) [inline source/kraken/configuration.cpp]: function is not called... you may want it later?

File: source/kraken/maintenance_worker.cpp (hunk 1 above)
Comment (xlqian) [inline source/kraken/maintenance_worker.cpp:72]: ok

File: source/navitia-proto [hunk 3]
```diff
@@ -1 +1 @@
-Subproject commit 8d116b329cf9ce5e173d0e5c046022c4804d9449
+Subproject commit d22f76ceb354fd21e64d53743c95f15154b06dec
```
Comment (pbougue) [inline source/navitia-proto]: one commit before master, should be good after rebase

File: source/kraken/configuration.cpp (hunk 2 above)
Comment (pbougue) [inline source/kraken/configuration.cpp]: I would name it `[is_|get_|]realtime_enabled`

File: source/kraken/maintenance_worker.cpp (hunk 1 above)
Comment (pbougue) [inline source/kraken/maintenance_worker.cpp:72]: minor: maybe add a comment on parameters (or name them, but too verbose)

File: source/kraken/configuration.cpp [hunk 4]
```diff
@@ -51,6 +51,8 @@ po::options_description get_options_description(const boost::optional<std::strin
          "name of the instance")
 
         ("GENERAL.nb_threads", po::value<int>()->default_value(1), "number of workers threads")
+        ("GENERAL.enable_realtime", po::value<bool>()->default_value(false), "enable loading of realtime data")
+        ("GENERAL.kirin_timeout", po::value<int>()->default_value(10000), "timeout in ms for loading realtime data from kirin")
```
Comment (pbougue) [inline source/kraken/configuration.cpp]: line length

**Comment by xlqian** [comment]
ok  for me

**Pull request merged**